src/nomad_simulations/schema_packages/basis_set.py

```diff
@@ -184,22 +184,444 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+            'j',
+            'k',
+            'l',
+            'sp',
+            'spd',
+            'spdf',
+        ),
+        description="""
+        Symbolic label for the **angular momentum** of this contracted function. 
+        Typical single-letter labels:
+          - 's' => L=0
+          - 'p' => L=1
+          - 'd' => L=2
+          - 'f' => L=3
+          - 'g' => L=4
+          - 'h', 'i', etc. => still higher angular momenta
+        Combined labels like 'sp' or 'spdf' indicate a **combined shell** in which 
+        multiple angular momenta share exponents. For example, in some older Pople 
+        basis sets, an 'sp' shell has an s- and p-type function sharing the same 
+        exponents but different contraction coefficients.
+        """,
+    )
+
+    n_primitive = Quantity(
+        type=np.int32,
+        description="""
+        Number of **primitive** functions in this contracted basis function. 
+        For example, in a contracted Gaussian-type orbital (GTO) approach, each basis 
+        function might be built from a sum of `n_primitive` Gaussians with different 
+        exponents, each scaled by a contraction coefficient.
+        """,
+    )
+
+    n_components = Quantity(type=np.int32)
+
+    contraction_coefficients = Quantity(
+        type=np.float32,
+        shape=['*'],
+        description="""
+        The **contraction coefficients** associated with each primitive exponent. 
+        In the simplest case (pure s- or p-function), this array has length 
+        equal to `n_primitive`. For combined shells (like 'sp'), the length 
+        might be `2 * n_primitive`, because you have separate coefficients 
+        for the s-part and the p-part.
+        """,
+    )
+
+    exponents = Quantity(
+        type=np.float32,
+        shape=['n_primitive'],
+        description="""
+        The **exponents** of each primitive basis function. 
+        In a Gaussian basis set (GTO), these are the alpha_i in 
+        exp(-alpha_i * r^2). In a Slater-type basis (STO), they'd be 
+        exp(-zeta_i * r). Typically sorted from largest to smallest.
+        """,
+    )
+
+    primitive_normalization = Quantity(
+        type=MEnum('L2', 'none', 'contracted', 'custom'),
+        default='L2',
+        description="""
+        Convention used for contraction coefficients:
+        • 'L2' - coefficients multiply L²-normalised primitives (Gaussian/BSE style).  
+        • 'none' - coefficients multiply unnormalised primitives (rare).  
+        • 'contracted' - primitives normalised, contracted function renormalised (GAMESS-style).  
+        • 'custom' - explicit factors are stored in `primitive_factor` / `shell_factor`.
+        """,
+    )
+
+    primitive_factor = Quantity(
+        type=np.float64,
+        shape=['n_primitive'],
+        description="""
+        **Extra scaling factors for each primitive** (dimensionless).
+
+        - Let ϕₖ(r) be the *analytic* primitive basis function (Gaussian or
```

Comment:
Can you check at some point if the greek letters are displayed properly in the GUI metainfo browser and DATA tab?